.github/workflows/unittests.yml

```diff
@@ -32,3 +32,18 @@ jobs:
       with:
         name: release
         path: release
+  go-mod:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.15.2'
```

Comment:
should we bump go version in go.mod as well? https://github.com/cilium/hubble/blob/f7d360b8921a5b958e07109983b8ca5e9cb40ab8/go.mod#L3

Reply:
I don't think we have any reason to unless we want to use Go features that are not available in go 1.14.